www/docs/en/dev/guide/platforms/electron/index.md

```diff
@@ -144,7 +144,7 @@ By default, the `width` is set to **800** and the `height` set to **600**. This
 #### How to make my window fullscreen?
 
 Using the `fullscreen` flag property, you can force the application to launch in fullscreen.
-
+1
```

Comment:
???

Reply:
That is weird. I scrolled though the commit and didn't see this haha.

Reply:
I don't know how this slipped through. I can make a cleanup PR.